frontend/integration-tests/tests/storage-dashboard.scenario.ts

```diff
@@ -0,0 +1,34 @@
+import { browser } from 'protractor';
+import { appHost } from '../protractor.conf';
```

Comment:
And this import path will change to `'@console/internal-integration-tests/protractor.conf'`